src/MovieSnapShot/MovieSnapShot.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import Header from "../Header/Header";
+import Footer from "../Footer/Footer";
+import MovieCard from "../MovieCard/MovieCard";
+import singleMovie from "../data/singleMovie.js";
+import "./MovieSnapShot.css";
+
+class MovieSnapShot extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      singleMovie: singleMovie.movie
+    }
+  }
+
+  render() {
+    const genres = this.state.singleMovie.genres.join(" | ");
+    return (
+      <div className="background-img" style={{backgroundImage: `url(${this.state.singleMovie.backdrop_path})`}}>
+        <Header onHomeClick={this.props.onHomeClick}/>
+        <main className="single-movie-container">
```

Comment:
I am still wondering if we can declare two main tags even if they are in separate components that render conditionally. Something we should ask about I think, I couldn't find any articles relating to it when doing research

Reply:
I will ask my mentor and see if he can point us down the right path. If not we should reach out to Leta or Scott.

Reply:
Ok, same here with my mentor